### ISSUE TITLE
Cannot open draw.io files from the right click context menu or from the flatpak run command

### ISSUE THREAD

**Issue by zeitue** [flathub/com.jgraph.drawio.desktop#130]
Cannot open draw.io files from the right click context menu or even from the flatpak run command
```bash
flatpak run com.jgraph.drawio.desktop ./diagram.drawio
```
The command line output only complains about not being able to access the system dbus, nothing about file access
```bash
[2:0530/221150.475665:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0530/221150.960777:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0530/221150.960809:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
```

![image](https://github.com/flathub/com.jgraph.drawio.desktop/assets/7822272/812ddae1-1864-4664-8bde-a47303650381)

The result is draw.io just loads with an untitled diagram.
I Tested the following:
- with both files in the $HOME directory and at system level `/mnt/share`
- from GUI and CLI
- gave access to system dbus
- gave filesystem host access
- tested on Manjaro
- tested on Kubuntu
- tested none flatpak version of draw.io

From this I found the none flatpak version does not have this issue and that nothing fixed the issue for the flatpak version.

**Comment by zeitue** [comment]
I've found that if the Draw.io flatpak is open already and you open a file with it in the file manager it works. So this is something to do with the first load.

**Comment by ghisvail** [comment]
Is the issue still happening on the most recent update?

**Comment by zeitue** [comment]
I tested version `22.1.16`,  and it still opens an untitled diagram over the file I clicked on.

**Comment by qgymib** [comment]
Yes i have the same problem, and the official `.appimage` release does not have this issue.
